internal/dcache/cluster_manager/cluster_manager.go

```diff
@@ -2054,17 +2054,23 @@ func (cmi *ClusterManager) joinMV(mvName string, mv dcache.MirroredVolume, reser
 
 		log.Debug("ClusterManagerImpl::joinMV: Joining MV %s with RV %s", mvName, rv.Name)
 
+		myNodeID, err1 := common.GetNodeUUID()
+		common.Assert(err1 == nil, err1)
+		common.Assert(common.IsValidUUID(myNodeID), myNodeID)
+
 		joinMvReq := &models.JoinMVRequest{
+			SenderNodeID: myNodeID,
```

Comment:
populating sender nodeid is a common operation that the rpc_client functions can do.. so callers need not worry about it.